### DISCUSSION TITLE
Spoilers

### DISCUSSION THREAD

**Discussion by JesusChapman** [Yisus7u7/termux-desktop-xfce#91] [Show and tell]
Here you will upload advances and functions that in the future may be imposed. 

You can freely give your opinion

**Comment by JesusChapman** [comment]
![Screenshot_20220220-234539](https://user-images.githubusercontent.com/64093255/154891189-63624240-9261-43ce-a663-efab6c1b40b2.png)

**Comment by Aridlvory** [comment]
After executing the installation command if there is an error it still says 'desktop successfully installed' 
It will be nice that if an error occurs break away from the loop and print an error message![Screenshot_20220224-191932_Termux.jpg](https://user-images.githubusercontent.com/78092537/155536699-902b9fc6-8a04-4739-979a-3245f10a4ec9.jpg)

**Reply by JesusChapman** [reply]
> After executing the installation command if there is an error it still says 'desktop successfully installed' It will be nice that if an error occurs break away from the loop and print an error message![Screenshot_20220224-191932_Termux.jpg](https://user-images.githubusercontent.com/78092537/155536699-902b9fc6-8a04-4739-979a-3245f10a4ec9.jpg)

It will be resolved in the next update
